api/src/wfl/server.clj

```diff
@@ -71,25 +71,33 @@
       wrap-internal-error
       (wrap-json-response {:pretty true})))
 
-(defn stfu-jetty
-  "Set up a stub logger to shut Jetty up."
-  []
-  (Log/setLog
-   (proxy [Logger] []
-     (debug ([thrown]) ([msg & args]))
-     (getLogger [name] this)
-     (getName [] "JettyStfu")
-     (ignore [ignored])
-     (info ([thrown]) ([msg & args]))
-     (isDebugEnabled [] false)
-     (setDebugEnabled [enabled])
-     (warn ([thrown]) ([msg & args])))))
+(defn ^:private start-workload-manager []
+  (letfn [(update-workloads! [])] ;; todo
+    (log/info "starting workload update loop")
+    (update-workloads!)
+    (future
+      (loop []
+        (update-workloads!)
+        (.sleep TimeUnit/SECONDS 20)
+        (recur)))))
```

Comment:
When does this loop exit?

Reply:
The loop doesn't end - it loops for the lifetime of the server.
The reason for the first call is I want to update the state of the wfl database BEFORE we start listening to requests.